WooCommerce/Classes/Model/Note+Woo.swift

```diff
@@ -66,3 +66,20 @@ private extension Note {
         static let emptyStar    = "\u{2606}"  // Unicode White Star ☆
     }
 }
+
+extension Note {
+    var orderID: Int? {
+        return meta.identifier(forKey: .order)
+    }
+
+    func markOrderNoteAsReadIfNeeded() {
```

Comment:
Since you opted for this approach, it would be helpful to add some comments to this method indicating that a network request will be performed.